TypeCobol/Compiler/Sql/Model/SqlObject.cs

```diff
@@ -18,6 +19,11 @@ public bool AcceptVisitor(ISqlVisitor visitor)
             visitor.EndSqlObject(this);
             return continueVisit;
         }
+        public virtual void Dump(TextWriter output, int indentLevel)
+        {
+            string indent = new string(' ', 2 * indentLevel);
+            output.Write(indent);
+        }
```

Comment:
This is useless, it will only write an indent string for sql objects without proper override.
I suggest:

```csharp
        public virtual void Dump(TextWriter output, int indentLevel)
        {
            string indent = new string(' ', 2 * indentLevel);
            output.Write(indent);
            output.WriteLine(GetType().FullName);
        }
```

Reply:
I insist, this method must be changed